### PR TITLE
chore: clean dhis-service poms

### PR DESCRIPTION
**Pull request by teleivo**
dhis-service pom declares dependencies although it does not have any dependencies. I assume it is so that its child modules all "inherit" these dependencies. Not every child module needs them. Each child module should be independent when it comes to its `<dependencies>`.

Cleanup ~15 services pom.xml files before removing it. This tackles the first 3 😅 

This is part of https://jira.dhis2.org/browse/TECH-801
I based it on https://github.com/dhis2/dhis2-core/pull/9372 as its helpful in analyzing dependencies. Please approve/merge https://github.com/dhis2/dhis2-core/pull/9372 before

### CODE REVIEW

File: dhis-2/dhis-services/dhis-service-schema/pom.xml [hunk 1]
```diff
@@ -64,6 +106,11 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.hisp.dhis</groupId>
+      <artifactId>dhis-support-test</artifactId>
```
Comment (enricocolasante) [inline dhis-2/dhis-services/dhis-service-schema/pom.xml:113]: Scope test?

File: dhis-2/dhis-services/dhis-service-setting/pom.xml [hunk 2]
```diff
@@ -29,22 +83,21 @@
...
-      <scope>compile</scope>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.jasypt</groupId>
-      <artifactId>jasypt</artifactId>
+      <groupId>org.hisp.dhis</groupId>
+      <artifactId>dhis-support-test</artifactId>
```
Comment (enricocolasante) [inline dhis-2/dhis-services/dhis-service-setting/pom.xml:100]: Same here?

**Review by enricocolasante — APPROVED** [review]
Just one doubt

File: dhis-2/dhis-services/dhis-service-schema/pom.xml (hunk 1 above)
Comment (teleivo) [reply dhis-2/dhis-services/dhis-service-schema/pom.xml:113]: set in parent

https://github.com/dhis2/dhis2-core/blob/92973bfa27592e13215ad38a10822b7d0509a0b2/dhis-2/pom.xml#L823-L828

but I'll make it explicit 👌🏻

**Review by ameenhere — APPROVED** [review]

**Pull request merged**